src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Command/RefundPlugin.php

```diff
@@ -24,6 +24,7 @@
 class RefundPlugin extends AbstractPlugin implements CommandByOrderInterface
 {
     /**
+     * {@inheritDoc}
      * Command which is executed per order basis
```

Comment:
- Sends "refund" request to AfterPay gateway, to refund payment for a specific order.
- Command which is executed per order basis.